api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/MessageRepository.java

```diff
@@ -17,8 +18,16 @@
 
     @Query("SELECT m "
             + "FROM Message m "
-            + "where m.messageType = :messageType "
+            + "where m.messageType = :messageType and m.messageTimestamp < :olderThan "
             + "order by m.messageTimestamp")
-    List<Message> findAllMessagesByTypeOrderedByDate(@Param("messageType") MessageType messageType, Pageable pageable);
+    List<Message> findAllMessagesByTypeOlderThan(
+            @Param("messageType") MessageType messageType,
+            @Param("olderThan") LocalDateTime olderThan,
+            Pageable pageable);
+
+    default List<Message> findAllMessagesDueForProcessingByType(
```

Comment:
I'm not too keen on the phrase `dueForProcessing`, its the kind of term that means something if you know what it means (if that makes sense). How about something like `findAllMessagesOlderThanNow` or `findAllPastMessages` as you can understand exactly what both of those mean, although they might sound a bit clunky, but they are understandable.